src/main/scala/dpla/ingestion3/model/ModelConverter.scala

```diff
@@ -14,13 +14,20 @@ import org.apache.spark.sql.Row
   * is responsible for converting a DplaMapData to a Row and represents the opposite of this transformation.
```

Comment:
Do you want to change the comment block above to say `OreAggregation` instead of `DplaMapData`?